bluetooth/idl/idlharness.tentative.https.window.js

```diff
@@ -5,15 +5,21 @@
 
 // https://webbluetoothcg.github.io/web-bluetooth/
 
-promise_test(async () => {
-  const srcs = ['web-bluetooth', 'dom', 'html', 'permissions'];
-  const [idl, dom, html, permissions] = await Promise.all(
-      srcs.map(i => fetch(`/interfaces/${i}.idl`).then(r => r.text())));
+idl_test(
+  ['web-bluetooth'],
+  ['dom', 'html', 'permissions'],
+  idl_array => {
+    try {
+      self.event = new BluetoothAdvertisingEvent('type');
+    } catch(e) {
+      // Surfaced in idlharness.js's test_object below.
```

Comment:
I don't see a `test_object` below.

If failure here is identified by `self.event` being `undefined`, please say that.

Reply:
`test_object` is called deeper in idlharness.js
https://github.com/web-platform-tests/wpt/blob/master/resources/idlharness.js#L2521

Updated the comment.